oxidation/parsec_cli/parsec_sequestre_dump/src/main.rs

```diff
@@ -0,0 +1,109 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+#[macro_use]
+extern crate diesel;
+
+mod model;
+mod schema;
+
+use clap::Parser;
+use diesel::{Connection, SqliteConnection};
+use std::path::PathBuf;
+
+use crate::model::Data;
+use parsec_api_crypto::SecretKey;
+
+#[derive(Parser, Debug)]
+#[clap(author, version, about, long_about = None)]
```

Comment:
```suggestion
#[clap(version, about)]
```
- not forced to set *long about* to `None`
- `Cargo.toml` don't have an `authors` section